cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -284,6 +284,11 @@ macro(swift_common_cxx_warnings)
   # Check for '-fapplication-extension'.  On OS X/iOS we wish to link all
   # dynamic libraries with this flag.
   check_cxx_compiler_flag("-fapplication-extension" CXX_SUPPORTS_FAPPLICATION_EXTENSION)
+
+  # Disable C4068: unknown pragma. This means that MSVC doesn't report hundreds of warnings across
+  # the repository for IDE features such as #pragma mark "Title".
+  check_cxx_compiler_flag("/wd4068" CXX_SUPPORTS_UNKNOWN_PRAGMA_FLAG)
+  append_if(CXX_SUPPORTS_UNKNOWN_PRAGMA_FLAG "/wd4068" CMAKE_CXX_FLAGS)
```

Comment:
So I am having issue with this argument when I build on Mac OS X; it seems to be triggering for me (https://bugs.swift.org/browse/SR-3952) I presume this change was intended to be a MSVC only thing? Should this somehow be gated by a platform guard or perhaps be of the `-wd4068` format?